modules/ting_search/tests/ting_search_unit.test

```diff
@@ -0,0 +1,131 @@
+<?php
+
+use Drupal\ding_test\DingUnitTestBase;
+use Prophecy\Argument;
+use Ting\Search\BooleanStatementGroup;
+use Ting\Search\BooleanStatementInterface;
+use Ting\Search\TingSearchCommonFields;
+use Ting\Search\TingSearchFieldFilter;
+use Ting\Search\TingSearchRequest;
+use Ting\Search\TingSearchStrategyInterface;
+use Ting\Search\UnsupportedSearchQueryException;
+
+class DingSearchUnitTest extends DingUnitTestBase {
+  public static function getInfo() {
+    return array(
+      'name' => 'Ting search Unittests',
+      'description' => 'Unit-tests that tests the ting search module.',
+      'group' => 'Ding! - Ting search unittest',
+      'dependencies' => array('ding_test'),
+    );
+  }
+
+  public function setUp() {
```

Comment:
Override function from parent and only calling patent has no effect and should be removed.

Reply:
This is true for all tests where this function is not used to alter the setup.

Reply:
:+1: